scripts/BONSpecialGenerator.swift

```diff
@@ -65,7 +66,7 @@ let charactersRequiringFormatStrings: Set<unichar> = [
 
 // These special characters are to be excluded from the human readable string dictionary.
 let charactersToExcludeFromHumanReadableStringDictionary: Set<unichar> = [
-    0x0020,
+    0x0020
```

Comment:
#nth don't kill this comma. The comma is there to prevent future additions to this array from messing with the revision history of previous lines. Kind of a moot point now, unless you rebase, but a good habit to get into.